package.json

```diff
@@ -132,7 +132,7 @@
     "vscode-languageserver-protocol": "3.15.3",
     "vscode-ws-jsonrpc": "^0.2.0",
     "walk": "^2.3.13",
-    "ws": "^5.0.0"
```

Comment:
Just bump up the explorook version

Reply:
no need because we didn't publish the last version yet.

Reply:
https://github.com/rookout/explorook/releases/latest